README.md

```diff
@@ -57,12 +57,11 @@ services:
     ports:
       - 8001:80
     volumes:
-      - ./nginx_site.conf:/etc/nginx/conf.d/default.conf
+      - ./nginx_site.conf:/etc/nginx/conf.d/default.conf:ro
+      - public:/opt/kimai/public:ro
```

Comment:
I am not a docker user, so I might be wrong, but Kimai writes avatar files to public/avatars/ so read-only would cause troubles here!?!

Reply:
No, the `fpm` container (php-handler) will write those files, not the `web` container.

Reply:
so I actually was wrong 😁 always good to learn!

Reply:
Look at this [nextcloud docker-compose sample](https://github.com/nextcloud/docker/blob/master/.examples/docker-compose/with-nginx-proxy/mariadb/fpm/docker-compose.yml#L31), it's done the same way.